docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -380,7 +384,11 @@ Reference](../../api/overview.md).
 | `http://tizen.org/feature/network.bluetooth.hid.device` | Specify this key, if the application requires the Bluetooth Human Interface Device (HID) device feature. | 3.0   |
 | `http://tizen.org/feature/network.bluetooth.le` | Specify this key, if the application requires the Bluetooth LE feature. | 2.3   |
 | `http://tizen.org/feature/network.bluetooth.le.5_0` | Specify this key, if the application requires the Bluetooth 5 feature. | 5.0   |
+| `http://tizen.org/feature/network.bluetooth.le.ipsp` | Specify this key, if the application requires the Bluetooth Internet Protocol Support Profile (IPSP). | 4.0   |
+| `http://tizen.org/feature/network.bluetooth.le.gatt.client` | Specify this key, if the application requires the Bluetooth Generic Attribute Profile (GATT) Client. | 4.0   |
+| `http://tizen.org/feature/network.bluetooth.le.gatt.server` | Specify this key, if the application requires the Bluetooth Generic Attribute Profile (GATT) Server. | 4.0   |
 | `http://tizen.org/feature/network.bluetooth.opp` | Specify this key, if the application requires the Bluetooth Object Push feature (OPP). | 2.3   |
+| `http://tizen.org/feature/network.bluetooth.oob` | Specify this key, if the application requires the Bluetooth Out Of Band (OOB). | 4.0   |
```

Comment:
Please make this small change:

Specify this key, if the application requires Bluetooth Out Of Band (OOB).

I removed 'the' as the sentence does not have a qualifier.

Reply:
done

Reply:
Please refer again, I see the usage of 'the' in the preview. Please do the needed.

Reply:
done